tests/runtests.py

```diff
@@ -17,6 +17,10 @@
     exclude_for_architectures
 ) # pylint: disable=import-error
 
+ARCH = subprocess.check_output("uname --processor".split()).strip().decode('utf-8')
```

Comment:
this is not necessarily set on every machine e.g. on mine:

```sh
$ uname --processor
unknown
```

Therefore `is_64b` returns `False` for me and e.g. I land in the "32bit-branches" in `test_func_heap` and `test_func_stack` which then fail.

:(

Reply:
how about `getconf LONG_BIT` instead?

Reply:
We need the architecture, not just checking if it's 64b.

Reply:
nit: use `"` not `'`